src/Workspaces/CSharp/Portable/Classification/Worker_Preprocesser.cs

```diff
@@ -324,6 +328,43 @@ private void ClassifyLoadDirective(LoadDirectiveTriviaSyntax node)
         ClassifyDirectiveTrivia(node);
     }
 
+    private void ClassifyIgnoredDirective(IgnoredDirectiveTriviaSyntax node)
+    {
+        AddClassification(node.HashToken, ClassificationTypeNames.PreprocessorKeyword);
+        AddClassification(node.ColonToken, ClassificationTypeNames.PreprocessorKeyword);
+
+        // The first part (separated by whitespace) of content is a "keyword", e.g., 'sdk' in '#:sdk Test'.
+        if (TryFindWhitespace(node.Content.ValueText, out var firstSpaceIndex))
+        {
+            var keywordSpan = new TextSpan(node.Content.SpanStart, firstSpaceIndex);
+            var stringLiteralSpan = new TextSpan(node.Content.SpanStart + firstSpaceIndex + 1, node.Content.Span.Length - firstSpaceIndex - 1);
+
+            AddClassification(keywordSpan, ClassificationTypeNames.PreprocessorKeyword);
+            AddClassification(stringLiteralSpan, ClassificationTypeNames.StringLiteral);
+        }
+        else
+        {
+            AddClassification(node.Content, ClassificationTypeNames.PreprocessorKeyword);
+        }
+
+        ClassifyDirectiveTrivia(node);
+
+        static bool TryFindWhitespace(string text, out int index)
+        {
+            for (var i = 0; i < text.Length; i++)
+            {
+                if (SyntaxFacts.IsWhitespace(text[i]))
+                {
+                    index = i;
+                    return true;
+                }
+            }
+
+            index = -1;
+            return false;
+        }
```

Comment:
```suggestion
```

Reply:
Are you suggesting to search just for space? Then having `#:sdk\tTest` wouldn't be colorized properly.

Reply:
I'm totally ok with that.

Reply:
or do `IndexOfAny([' ', '\t'])` if you do want to support that.  i'm fine with both.